powerauth-java-server/src/main/java/com/wultra/security/powerauth/app/server/service/util/jwt/JWSActivationSigner.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2025 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.wultra.security.powerauth.app.server.service.util.jwt;
+
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jose.JWSAlgorithm;
+import com.nimbusds.jose.JWSHeader;
+import com.nimbusds.jose.JWSSigner;
+import com.nimbusds.jose.jca.JCAContext;
+import com.nimbusds.jose.util.Base64URL;
+import com.wultra.security.powerauth.app.server.database.model.KeyType;
+import com.wultra.security.powerauth.app.server.database.model.entity.ActivationRecordEntity;
+import com.wultra.security.powerauth.app.server.service.crypto.CryptographyService;
+
+import java.util.Set;
+
+/**
+ * Signer for JWT for PowerAuth activations.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+public class JWSActivationSigner implements JWSSigner {
+
+    public static final String JWT_ALGORITHM_NAME_ES256 = "ES256";
+    public static final String JWT_ALGORITHM_NAME_ES384 = "ES384";
+    public static final String JWT_ALGORITHM_NAME_MLDSA_65 = "MLDSA-65";
```

Comment:
It seems `ML-DSA-65` will be standardised. https://www.ietf.org/archive/id/draft-ietf-cose-dilithium-04.html#name-ml-dsa-algorithms
```suggestion
    public static final String JWT_ALGORITHM_NAME_MLDSA_65 = "ML-DSA-65";
```